### PR TITLE
[UI-Refactor] Navigation: fix minor bugs in nav

### PR DESCRIPTION
**Pull request by abhinavkrin**
Signed-off-by: Abhinav Kumar <abhinav@avitechlab.com>

**Description**
This PR fixes minor bugs in nav.
1. Navbar gets hidden by header when the navbar is collapsed. The issue was due to the zIndex of the header being greater than that of the navbar.
2. App crashed when clicking on the navbar collapse icon button. This issue occurred because when in collapsed mode the navbar tried to render the `version` object instead of `version.build` and as we know react cant render javascript objects. This must be a typo.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Navbar/Navbar.js
Comment (abhinavkrin) [inline ui/components/Navbar/Navbar.js:414]: The `build` property of the `serverVersion` object has to be rendered.

File: ui/components/Navbar/Navbar.js
Comment (abhinavkrin) [inline ui/components/Navbar/Navbar.js:414]: Rendering object `serverVersion`. This was the reason for error.

File: ui/components/Navbar/Navbar.js
Comment (abhinavkrin) [inline ui/components/Navbar/Navbar.js:372]: Increased z-index of the navbar to prevent getting hidden by header.

**Comment by leecalcote** [comment]
Please label with `ui-refactor` and component/ui`

**Comment by abhinavkrin** [comment]
> Please label with `ui-refactor` and `component/ui`

I would need write access to do that.

File: ui/components/Navbar/Navbar.js
Comment (YashKamboj) [reply ui/components/Navbar/Navbar.js:414]: Oh , i can't beleive i missed that!! 😅

**Review by YashKamboj — APPROVED** [review]
LGTM!!!! Great Work @abhinavkrin ! 🚀

File: ui/components/Navbar/Navbar.js
Comment (sudo-NithishKarthik) [reply ui/components/Navbar/Navbar.js:414]: nice catch @abhinavkrin.

**Comment by sudo-NithishKarthik** [comment]
> @abhinavkrin This PR shows a lot of changes , i guess this is due to prettier , it gets really difficult review, can you please take care of this in your next pr

@abhinavkrin , this shows that your dev environment is not properly configured with our settings. Please make you do that and update the PR. We shouldn't merge it unless it is done.

**Comment by abhinavkrin** [comment]
Hi @sudo-NithishKarthik @YashKamboj. It was due to the lines ending with CRLF. Converted it back to LF.

File: ui/components/Navbar/Navbar.js
Comment (YashKamboj) [inline ui/components/Navbar/Navbar.js:372]: Can you avoid inline css here @abhinavkrin

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: ui/components/Navbar/Navbar.js
Comment (abhinavkrin) [reply ui/components/Navbar/Navbar.js]: Done!

**Review by YashKamboj — APPROVED** [review]
LGTM!!!!

**Pull request merged**
